packages/@react-aria/collections/src/Document.ts

```diff
@@ -311,14 +311,14 @@ export class ElementNode<T> extends BaseNode<T> {
     this.ownerDocument.queueUpdate();
   }
 
-  get style() {
+  get style(): CSSProperties {
     return {};
   }
 
-  hasAttribute() {}
-  setAttribute() {}
-  setAttributeNS() {}
-  removeAttribute() {}
+  hasAttribute(): void {}
+  setAttribute(): void {}
+  setAttributeNS(): void {}
+  removeAttribute(): void {}
```

Comment:
ugh, do we really have to annotate functions that don't return anything?

Reply:
I tried it with the typescript features on and the eslint rule off, these appear to be necessary.
```
Method must have an explicit return type annotation with --isolatedDeclarations.
```